### PR TITLE
Kgaut chandima file edits a1a

### PR DESCRIPTION
**Pull request by kgauthreaux**

### CODE REVIEW

**Review by hewanadun — COMMENTED** [review]
@kgauthreaux 
Noticed that comp_forms, comp_vars columns filled with "n/a" in form A1a checks but they left blank in form A1 checks. 
It would be easy for processing if those columns left blank when there are no multiple forms/vars involved in the check. But, if most of the forms already have "n/a" and if it is easy to change all forms to "n/a" that would work too, just need to be consistent across all forms.

File: forms/uds/a1a/form_a1a_ivp_error_checks_p.csv [hunk 1]
```diff
@@ -0,0 +1,12 @@
+error_code,error_no,error_type,form_name,packet,var_name,check_type,test_name,short_desc,full_desc,test_logic,comp_forms,comp_vars,do_in_redcap,in_prev_versions,questions
+a1a-ivp-p-1001,1001,Alert,a1a,I,TRANSPROB,Plausibility,Consistency in transportation access across forms,If TRANSPROB or TRANSWORRY = 1 (often) and Form A1 TRANSPOR = 4 (never limits me),If Q3. TRANSPROB (Transportation problems in past month?) or Q4. TRANSWORRY (Transportation worries in past month?) = 1 (often) and Form A1 Q23. TRANSPOR (lack of transportation) = 4 (never limits me),If (TRANSPROB=1 or TRANSWORRY=1) and TRANSPOR=4,a1,"TRANSPROB, TRANSWORRY, TRANSPOR",No,No,
```
Comment (hewanadun) [inline forms/uds/a1a/form_a1a_ivp_error_checks_p.csv:2]: Form A1 doesn't have a variable named TRANSPOR? A1. 23 is a different question.

File: forms/uds/a1a/form_a1a_ivp_error_checks_p.csv [hunk 2]
```diff
@@ -0,0 +1,12 @@
+error_code,error_no,error_type,form_name,packet,var_name,check_type,test_name,short_desc,full_desc,test_logic,comp_forms,comp_vars,do_in_redcap,in_prev_versions,questions
+a1a-ivp-p-1001,1001,Alert,a1a,I,TRANSPROB,Plausibility,Consistency in transportation access across forms,If TRANSPROB or TRANSWORRY = 1 (often) and Form A1 TRANSPOR = 4 (never limits me),If Q3. TRANSPROB (Transportation problems in past month?) or Q4. TRANSWORRY (Transportation worries in past month?) = 1 (often) and Form A1 Q23. TRANSPOR (lack of transportation) = 4 (never limits me),If (TRANSPROB=1 or TRANSWORRY=1) and TRANSPOR=4,a1,"TRANSPROB, TRANSWORRY, TRANSPOR",No,No,
+a1a-ivp-p-1002,1002,Alert,a1a,I,TRANSPROB,Plausibility,Consistency in transportation access across forms,If TRANSPROB or TRANSWORRY = 3 (never) and Form A1 TRANSPOR is 1 (always limits me),If Q3. TRANSPROB (Transportation problems in past month?) or Q4. TRANSWORRY (Transportation worries in past month?) = 3 (often) and Form A1 Q23. TRANSPOR (lack of transportation) = 1 (always limits me),If (TRANSPROB=3 or TRANSWORRY=3) and TRANSPOR=1,a1,"TRANSPROB, TRANSWORRY, TRANSPOR",No,No,
```
Comment (hewanadun) [inline forms/uds/a1a/form_a1a_ivp_error_checks_p.csv:3]: Same as above, Form A1 doesn't have a variable named TRANSPOR? A1. 23 is a different question.

File: forms/uds/a1a/form_a1a_ivp_error_checks_p.csv [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+a1a-ivp-p-1002,1002,Alert,a1a,I,TRANSPROB,Plausibility,Consistency in transportation access across forms,If TRANSPROB or TRANSWORRY = 3 (never) and Form A1 TRANSPOR is 1 (always limits me),If Q3. TRANSPROB (Transportation problems in past month?) or Q4. TRANSWORRY (Transportation worries in past month?) = 3 (often) and Form A1 Q23. TRANSPOR (lack of transportation) = 1 (always limits me),If (TRANSPROB=3 or TRANSWORRY=3) and TRANSPOR=1,a1,"TRANSPROB, TRANSWORRY, TRANSPOR",No,No,
+a1a-ivp-p-1003,1003,Alert,a1a,I,BILLPAY,Plausibility,Consistency in income across forms,"IF BILLPAY = 4 or 5 (very or extremely), and Form A1 INCOME 3 or 4 (have excess income)","IF Q7. BILLPAY (Difficulty with monthly payments?) = 4 or 5 (very or extremely), and Form A1 Q18. INCOME (household income) = 3 or 4 (have excess income)","If BILLPAY in (4,5) and INCOME in (3,4)",a1,"BILLPAY, INCOME",No,No,
```
Comment (hewanadun) [inline forms/uds/a1a/form_a1a_ivp_error_checks_p.csv:4]: Form A1 doesn't have a variable named INCOME? A1. 18 is a different question.

File: forms/uds/a1a/form_a1a_ivp_error_checks_p.csv [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+a1a-ivp-p-1003,1003,Alert,a1a,I,BILLPAY,Plausibility,Consistency in income across forms,"IF BILLPAY = 4 or 5 (very or extremely), and Form A1 INCOME 3 or 4 (have excess income)","IF Q7. BILLPAY (Difficulty with monthly payments?) = 4 or 5 (very or extremely), and Form A1 Q18. INCOME (household income) = 3 or 4 (have excess income)","If BILLPAY in (4,5) and INCOME in (3,4)",a1,"BILLPAY, INCOME",No,No,
+a1a-ivp-p-1004,1004,Alert,a1a,I,BILLPAY,Plausibility,Consistency in income across forms,"IF BILLPAY = 1 (not at all), and Form A1 INCOME = 1 (insufficient to meet needs)","IF Q7. BILLPAY (Difficulty with monthly payments?) = 1 (not at all), and Form A1 Q18. INCOME (household income) = 1 (insufficient to meet needs)",If BILLPAY=1  and INCOME=1,a1,"BILLPAY, INCOME",No,No,
```
Comment (hewanadun) [inline forms/uds/a1a/form_a1a_ivp_error_checks_p.csv:5]: Same as above, Form A1 doesn't have a variable named INCOME? A1. 18 is a different question.

File: forms/uds/a1a/form_a1a_ivp_error_checks_p.csv [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+a1a-ivp-p-1004,1004,Alert,a1a,I,BILLPAY,Plausibility,Consistency in income across forms,"IF BILLPAY = 1 (not at all), and Form A1 INCOME = 1 (insufficient to meet needs)","IF Q7. BILLPAY (Difficulty with monthly payments?) = 1 (not at all), and Form A1 Q18. INCOME (household income) = 1 (insufficient to meet needs)",If BILLPAY=1  and INCOME=1,a1,"BILLPAY, INCOME",No,No,
+a1a-ivp-p-1005,1005,Alert,a1a,I,PARENTCOMM,Plausibility,Consisitency in family/friend contact across forms,"If PARENTCOMM, CHILDCOMM, or FRIENDCOMM are >2 (several times a month or more) and Form A1 CONTACT = 1 (none)","If Q21. PARENTCOMM (Frequency of communication with parents?), Q22. CHILDCOMM (Frequency of communication with children?), or Q23. FRIENDCOMM (Frequency of communication with friends?) are >2 (several times a month or more) and Form A1 Q24. CONTACT (close contact with friends/relatives) =1 (none)",If (PARENTCOMM >2 or CHILDCOMM >2 or FRIENDCOMM >2) and CONTACT=1,a1,"PARENTCOMM, CHILDCOMM, FRIENDCOMM, CONTACT",No,No,
```
Comment (hewanadun) [inline forms/uds/a1a/form_a1a_ivp_error_checks_p.csv:6]: Form A1 doesn't have a variable named CONTACT? A1. 24 is a different question.

**Review by hewanadun — CHANGES_REQUESTED** [review]
@kgauthreaux please check my comments, specially in plausibility checks with A1 form, I cannot find those variables in the A1 form (In A1 pdf or questions and vars).

File: forms/uds/a1a/form_a1a_ivp_error_checks_p.csv (hunk 5 above)
Comment (kgauthreaux) [reply forms/uds/a1a/form_a1a_ivp_error_checks_p.csv:6]: @hewanadun okay for these above these were old A1 variables that got deleted and the error checks were never updated! I deleted them now.

**Comment by kgauthreaux** [comment]
@hewanadun the edits you requested have been made to the a1a error check files!

**Review by hewanadun — APPROVED** [review]
Looks good.

**Pull request merged**